lib/mongo/protocol/utils.ex

```diff
@@ -109,8 +109,11 @@ defmodule Mongo.Protocol.Utils do
     {:disconnect, error, s}
   end
 
-  def namespace(coll, s),
+  def namespace(coll, s, database \\ nil)
```

Comment:
Maybe we should make the last argument explicit so that we don't forget to pass it somewhere?